Assignment5/Q1.py

```diff
@@ -0,0 +1,150 @@
+# coding=utf-8
+"""The solution presented below takes θ(N+M) runtime where N is the number of characters in dictionary words (sum of
```

Comment:
You are right that big theta provide more information and in this case it is not that difficult to compute, but it is often not required.